modules/programs/tmux.nix

```diff
@@ -48,6 +174,29 @@ in
         '';
       };
 
+      shortcut = mkOption {
+        default = defaultShortcut;
+        example = "a";
+        type = types.str;
+        description = "Ctrl following by this key is used as the main shortcut.";
+      };
+
+      terminal = mkOption {
+        default = defaultTerminal;
+        example = "screen-256color";
+        type = types.str;
+        description = "Set the $TERM variable.";
+      };
+
+      secureSocket = mkOption {
+        default = true;
+        type = types.bool;
+        description = ''
+          Store tmux socket under /run, which is more secure than /tmp, but as a
```

Comment:
```xml
Store tmux socket under <filename>/run</filename>, which is more
secure than <filename>/tmp</filename>, but as a
```

Reply:
Done.

PS: Have you tried pressing `Ctrl+g` to suggest changes?

![screenshot_5](https://user-images.githubusercontent.com/87115/52681580-212b6980-2ef1-11e9-97bb-b4e4b1e955aa.png)